person.rb

```diff
@@ -1,17 +1,18 @@
 require './corrector'
 
 class Person
+  attr_accessor :name, :age
+  attr_reader :rentals, :id
+
```

Comment:
- [ ] Let us add :rentals inside the `attr_accessor`

Reply:
- [ ] Good job here so far @pasytchangwa but you should implement a method for adding a `rental` here and `person` itself should be sent as a parameter to create a new rental